tests/integration/collective/src/collective_test.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include "collective_cffi.h"
+#include "legate_library.h"
+
+namespace collective {
+
+extern Legion::Logger logger;
```

Comment:
if this logger isn't used, please remove it. I feel we can remove this header file.